### PR TITLE
Update CITATION.cff with contributors for 0.5.0

### PR DESCRIPTION
**Pull request by jni**
Thanks to @Czaki for reminding me that there's no need to do this manually when he created a [script](https://github.com/napari/napari-release-tools/blob/main/find_contributors_without_citation.py) to get this info. 😅

**Edit:** categorising into not responded/added/declined:

Not yet responded:
@AmirAflak @DanGonite57 @mstabrin 

Added:
@AndrewAnnex @MarchisLost @aeisenbarth @imagejan @jamesyan-git @jo-mueller @maweigert @odinsbane @palec87 @perlman

Declined:
@stefanv

Thank you for contributing to napari! If you would like to be included in the CITATION.cff file for the next release, please comment by filling in the following template (in a code block) and I'll add your response to the file:

```
- given-names:
  family-names:
  affiliation:
  orcid: (optional, format as url: https://orcid.org/0000-0000-0000-0000)
  alias: (gh handle)
```

### CODE REVIEW

**Comment by perlman** [comment]
```
- given-names: Eric
  family-names: Perlman
  affiliation: Yikes LLC
  orcid: 0000-0001-5542-1302
  alias: perlman

```

**Comment by AndrewAnnex** [comment]
```
- given-names: Andrew
  family-names: Annex
  affiliation: SETI Institute/NASA ARC
  orcid: 0000-0002-0253-2313
  alias:  AndrewAnnex
```

File: CITATION.cff [hunk 1]
```diff
@@ -255,6 +255,11 @@ authors:
 - given-names: Konstantin
   family-names: Sofiiuk
   alias: ksofiyuk
+- given-names: Johannes
+  family-names: Soltwedel
+  affiliation: DFG cluster of excellence 'Physics of Life', TU Dresden
+  orcid: 0000-0003-1273-2412
```
Comment (Czaki) [inline CITATION.cff]:
```suggestion
  orcid: https://orcid.org/0000-0003-1273-2412
```

Validation workflow is working :)

Comment (jni) [reply CITATION.cff]: Nice! So annoying though that it needs the URL. I'll update my template above.

Comment (Czaki) [reply CITATION.cff]: Lack of URL results in failure to deploy of release to zenodo.

Comment (jni) [reply CITATION.cff]: Yes, I remember now 😅, I'm just saying, that seems like a failure in Zenodo/the cff format.

**Comment by MarchisLost** [comment]
`- given-names: David
  family-names: Pinto 
  affiliation:
  orcid: 
  alias: MarchIsLost`

**Comment by imagejan** [comment]
```
- given-names: Jan
  family-names: Eglinger
  affiliation: Friedrich Miescher Institute for Biomedical Research (FMI), Basel (Switzerland)
  orcid: https://orcid.org/0000-0001-7234-1435
  alias: imagejan
```

**Comment by aeisenbarth** [comment]
```
- given-names: Andreas
  family-names: Eisenbarth
  affiliation: EMBL Heidelberg, Germany
  orcid: https://orcid.org/0000-0002-1113-9556
  alias: aeisenbarth
```

**Comment by palec87** [comment]
`- given-names: David
  family-names: Palecek
  affiliation: Algarve Centre of Marine Sciences (CCMAR)
  orcid: https://orcid.org/0009-0003-9328-8540
  alias: palec87
`

**Comment by jamesyan-git** [comment]
```
- given-names: James
  family-names: Ryan
  affiliation: 
  orcid: 
  alias: jamesyan-git
```

**Comment by maweigert** [comment]
- given-names: Martin
  family-names: Weigert
  affiliation: TU-Dresden / EPFL
  orcid: https://orcid.org/0000-0002-7780-9057
  alias: maweigert

File: CITATION.cff [hunk 2]
```diff
@@ -227,6 +252,9 @@ authors:
   affiliation: Brown University
   orcid: https://orcid.org/0000-0003-4501-5428
   alias: kir0ul
+- given-names: David
+  family-names: Pinto
+  alias: MarchIsLost
```
Comment (Czaki) [inline CITATION.cff]:
```suggestion
  alias: MarchisLost
```

**Comment by jni** [comment]
@Czaki let's merge this today/tomorrow to have this in by the alpha, and make a new PR for remaining folks and docs folks.

**Comment by stefanv** [comment]
Re: my declined, I declined because I do not believe that I have made a substantial enough contribution to Napari to be credited as an author. I hope that changes at some point, but that is the status quo.

**Comment by Czaki** [comment]
> Re: my declined, I declined because I do not believe that I have made a substantial enough contribution to Napari to be credited as an author. I hope that changes at some point, but that is the status quo.

Maybe we should add `.zenodo.json` that allow to define authors and contributors (see [example](https://github.com/4DNucleome/PartSeg/blob/develop/.zenodo.json)) and provide a clear threshold how to jump from one category to another?

**Comment by palec87** [comment]
> > Re: my declined, I declined because I do not believe that I have made a substantial enough contribution to Napari to be credited as an author. I hope that changes at some point, but that is the status quo.
> 
> Maybe we should add `.zenodo.json` that allow to define authors and contributors (see [example](https://github.com/4DNucleome/PartSeg/blob/develop/.zenodo.json)) and provide a clear threshold how to jump from one category to another?

@Czaki, I agree. My contribution was miniscule as well, so I think it is appropriate to distinguish levels of contributions.

**Comment by jni** [comment]
> a clear threshold how to jump from one category to another?

I have yet to find a clear, objective metric that can't be gamed or the target of gaming — or even just reasonable disagreement among informed participants. I think it is a fool's errand to try to create a perfectly fair system.

Which is to say, I mostly object to the phrase "clear threshold". "Rough guidelines", I think, is a better target here. (Which yes, we probably should work towards. But also, zenodo records are still considered "toy" credit in academic circles, so I'm more than happy to err on the side of permissiveness for this. When the paper comes around, we can have harder discussions. 😅)

In the meantime, @Czaki, could you please approve so we can merge and move to the next PR?

**Review by Czaki — APPROVED** [review]

**Pull request merged**